packages/w3c-webdriver/.gitignore

```diff
@@ -0,0 +1,2 @@
+README.md
```

Comment:
are you sure you want to have these in gitignore?

Reply:
Yes because they are copied from the root

Reply:
In the root they are not ignored